purchase_order_line_sequence/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================================
+Purchase order lines with sequence number
+=========================================
+
+Provides a new field sequence on the purchase order, which allows to manage the order of its purchase order lines.
+The sequence then is propagated to the Invoice lines and the Stock Moves.
```

Comment:
The sequence is not propagated to the stock moves, only to the invoice.

Reply:
You should see in the <code>Initial Demand</code> tab